app/webpacker/components/TeamsCommitteesCouncils/index.jsx

```diff
@@ -46,12 +46,12 @@ export default function TeamsCommitteesCouncils({ canViewPastRoles }) {
   const hashIsValid = Boolean(activeGroup);
 
   useEffect(
-    function sanitizeHash() {
```

Comment:
The linter doesn't allow named functions in effects? Naming the effect is useful.